code/game/objects/items.dm

```diff
@@ -116,6 +116,8 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	if(GLOB.rpg_loot_items)
 		rpg_loot = new(src)
 
+	GLOB.all_items_list[src] = TRUE

```

Comment:
no lol.
look at ```V``` in my example, it's the thing being added.

you want to add ```src``` to the list, what on earth is adding total_number_of_item ```TRUE```s going to do for you?